account-kit/smart-contracts/src/ma-v2/modules/paymaster-guard-module/module.ts

```diff
@@ -0,0 +1,49 @@
+import { encodeAbiParameters, type Address, type Hex } from "viem";
+import { paymasterGuardModuleAbi } from "./abis/paymasterGuardModuleAbi.js";
+
+const addresses = {
+  default: "0x00", //dummy address, replace with deployed module address?
+} as Record<number | "default", Address>;
+
+const meta = {
+  name: "PaymasterGuardModule",
+  version: "alpha.1",
+  addresses,
+};
+
+export const PaymasterGuardModule = {
+  meta,
+  abi: paymasterGuardModuleAbi,
+  encodeOnInstallData: (args: {
+    entityId: number;
+    paymaster: Address;
+  }): Hex => {
+    const { entityId, paymaster } = args;
+    return encodeAbiParameters(
+      [
+        {
+          type: "uint32",
+          value: entityId,
```

Comment:
`value` isn't a field in the `encodeAbiParameters` API. Each of these elements should have the fields `type` and optionally `name`, where I think `name` is only used for error messages. See [the docs](https://viem.sh/docs/abi/encodeAbiParameters).